Parsers/ASimProcessEvent/Parsers/ASimProcessEvent.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
```

Comment:
we would need to update this file as well:
https://github.com/Azure/Azure-Sentinel/blob/23b502e285b220f319d6ebea53118a10e8025c79/Parsers/ASimProcessEvent/Parsers/ASimProcessEventCreate.yaml

Reply:
DONE

Reply:
not visible. Please validate

Reply:
added